lib/compat/wordpress-6.1/class-wp-theme-json-resolver-6-1.php

```diff
@@ -25,6 +25,19 @@ class WP_Theme_JSON_Resolver_6_1 extends WP_Theme_JSON_Resolver_6_0 {
 	 */
 	protected static $core = null;
 
+	/**
+	 * Container for keep track of registered blocks.
+	 *
+	 * @since 6.1.0
+	 * @var array
+	 */
+	protected static $blocks_cache = array(
+		'core'   => array(),
+		'blocks' => array(),
+		'theme'  => array(),
+		'user'   => array(),
+	);
+
```

Comment:
```suggestion
```

Not needed, as this class will inherit it from Core's `WP_Theme_JSON_Resolver` class.

Reply:
After talking with Jonny, this code is needed for sites that are running < WP 6.1 with the plugin.

Also need:

```php
protected static $blocks = null;
```

this too.



Sorry for the confusion.